Refactor scripts (#1160)
ericcornelissen/shescape: CONTRIBUTING.md

```

By default, fuzzing goes on forever - until a problem is found. You can change

> this by using `FUZZ_TIME` environment variable. This allows you to specify an
> integer number of seconds to fuzz. For example, to fuzz 10 seconds use:

Author: ```suggestion
this using the `FUZZ_TIME` environment variable. This enables you to specify how
long to fuzz as an integer number of seconds. For example, to fuzz 10 seconds:
```

Author: Resolved with 29516dd9229d2fc2b948c585542ec26d0e7857e4
